### PR TITLE
Hybrid Graph + Data page on the watch

### PR DESCRIPTION
**Pull request by bharat**
This change makes the graph smaller so that you can see the first line of the data display at the bottom of the graph. This hybrid interface lets folks see one valuable piece of data, without compromising the simplicity of the watch interface, overall.

This is what it looks like when you are scrolled to the top. You can see the whole graph, and the "Active Insulin" value:
img_2298

When you scroll down, you can see all of the data and a tiny bit of the bottom of the graph at the top of the screen:
img_2299

### CODE REVIEW

**Comment by ps2** [comment]
img_0381
The two white boxes are the same height.  The top of the graph should align with the bottom of the bottom white box.

**Comment by bharat** [comment]
I just took a screenshot from my watch and applied matching rectangles to it:
compare

I'm using a 42mm watch.

File: WatchApp Extension/Controllers/ChartHUDController.swift [hunk 1]
```diff
@@ -122,34 +122,29 @@ final class ChartHUDController: HUDInterfaceController, WKCrownDelegate {
             return numberFormatter
         }()
 
-        iobLabel.setHidden(true)
         if let activeInsulin = activeContext.iob, let valueStr = insulinFormatter.string(from: activeInsulin) {
             iobLabel.setText(String(format: NSLocalizedString(
-                    "IOB %1$@ U",
+                    "%1$@ U",
```
Comment (ps2) [inline WatchApp Extension/Controllers/ChartHUDController.swift]: Now that that IOB and COB are gone, you can use LoopKit.QuantityFormatter

File: WatchApp Extension/Extensions/NSUserDefaults+WatchApp.swift [hunk 2]
```diff
@@ -12,6 +12,7 @@ import Foundation
 extension UserDefaults {
     private enum Key: String {
         case StartOnChartPage = "com.loudnate.Naterade.StartOnChartPage"
+        case VisibleDuration = "com.loudnate.Naterade.VisibleDuration"
```
Comment (ps2) [inline WatchApp Extension/Extensions/NSUserDefaults+WatchApp.swift]: StartOnChartPage -> startOnChartPage, and VisibleDuration ->  visibleDuration.

File: WatchApp Extension/Scenes/GlucoseChartScene.swift [hunk 3]
```diff
@@ -204,18 +205,19 @@ class GlucoseChartScene: SKScene {
...
         let height: CGFloat
 
-        switch width {
-        case let x where x < 150:  // 38mm
-            height = 68
-        case let x where x > 180:  // 44mm
-            height = 106
-        default:
-            height = 86
+        switch screen.width {
```
Comment (ps2) [inline WatchApp Extension/Scenes/GlucoseChartScene.swift]: This should revert to the previous switch statement, which is based on how asset catalogs work; handle the 38 and the 44 as custom, and default to 42/40 which are nearly the same screen size.

Also the added comments are wrong; "162x197 px" is points, not pixels (the screen is @2x)

**Comment by ps2** [comment]
img_0381

text insets should align to the app title label, which is different on Series 4 than on Series 0-3

**Comment by ps2** [comment]
> That's odd. Is that image one that I provided for you? Or is it one that you built yourself? I'm wondering if I mixed it up and gave you the wrong screenshot...

I'm using a 44mm watch, and using a clean checkout of your `dev-watch-hybrid2` branch.  I did a clean build, and removed and reinstalled the watch app, and I see the same spacing as in my screenshot.

**Comment by ps2** [comment]
This work is being merged in #863

**Pull request closed without merging**